lib/js/components/Buttons/Button/Button.stories.ts

```diff
@@ -0,0 +1,104 @@
+// Button.stories.ts
+
+import Button from './Button.vue';
+import {
+	BUTTON_SIZES,
+	BUTTON_TYPES,
+	BUTTON_COLORS,
+	BUTTON_STATES,
+	BUTTON_RADIUSES,
+	BUTTON_ELEVATIONS,
+} from './Button.consts';
+import { ICONS } from '../../Icon';
+
+import { Meta, Story } from '@storybook/vue';
+
+export default {
+	title: 'Components/Buttons/Button',
+	component: Button,
+} as Meta<typeof Button>;
+
+const Template: Story<{
```

Comment:
I see in docs that Storybook recommends `StoryFn()`: https://storybook.js.org/docs/vue/writing-stories/introduction
What was the reason to not follow that rule?